static/css/site.css

```diff
@@ -118,6 +118,11 @@ section.content {
   margin: 1em 0;
 }
 
+.former .card {
+  zoom: 0.65;
+  width: 20em;
+}
+
```

Comment:
I would avoid tweaking CSS if possible, that would mean not adding this…

Reply:
heh, clearly wasn't thinking much here 😀